botfront/imports/ui/components/forms/graph/SlotChoiceModal.jsx

```diff
@@ -173,6 +173,7 @@ const SlotChoiceModal = (props) => {
                 'float',
```

Comment:
Why should we not be able to set value for all slot types?

Also, line 128 (I wasn't able to comment inline) seems to have been wrong for a while too: it should be possible to ask a question to fill an unfeaturized slot.

Reply:
given you can only choose between null and set for text slots and null and 1 for float slots they don't seem like they would provide functionality that a bool slot does not have. Very possibly wrong on this one but it has been this way since the set slot feature was created.

Reply:
not sure on the unfeaturized slot point, asked in secret-botfront